### PR TITLE
[1578] Minio Deployment Config 

### PR DESCRIPTION
**Pull request by AlexZorkin**

### CODE REVIEW

**Comment by DerekRoberts** [comment]
[EPIC#Pipeline | Create Minio Template and Deployment |2| !! [not before cutover]](https://trello.com/c/saGEjcQA/1578-epicpipeline-create-minio-template-and-deployment-2-not-before-cutover)

File: openshift/minio.bc.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: gwells-backend-bc
```
Comment (stephenhillier) [inline openshift/minio.bc.yaml]:
```suggestion
  name: gwells-minio-bc
```

File: openshift/minio.bc.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  value: "latest"
+- name: SRC_REPO_URL
+  value: "https://github.com/BCDevOps/minio-openshift"
+- name: SRC_REPO_BRANCH
+  value: "master"
+objects:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: "${DEST_IMG_NAME}"
```
Comment (stephenhillier) [inline openshift/minio.bc.yaml]:
```suggestion
    name: "${DEST_IMG_NAME}"
    labels:
      app: "${NAME}"
```

File: openshift/minio.dc.yaml [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      resources:
+        requests:
+          storage: ${DEST_PVC_SIZE}
+      storageClassName: ${DEST_PVC_CLASS}
+  - apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      finalizers:
+      - kubernetes.io/pvc-protection
+      name: "${NAME}-data-vol"
```
Comment (stephenhillier) [inline openshift/minio.dc.yaml]:
```suggestion
      name: "${NAME}-data-vol"
      labels:
        app: "${NAME}"
```

File: openshift/minio.dc.yaml [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  - name: "DEST_PVC_ACCESS"
+    displayName: "PVC access mode"
+    value: "ReadWriteOnce"
+objects:
+  - apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      finalizers:
+      - kubernetes.io/pvc-protection
+      name: "${NAME}-config-vol"
```
Comment (stephenhillier) [inline openshift/minio.dc.yaml]:
```suggestion
      name: "${NAME}-config-vol"
      labels:
        app: "${NAME}"
```

**Pull request merged**